### ISSUE TITLE
How to highlight border of a sprite in cocos2d-x

### ISSUE THREAD

**Issue by Ravi1092** [fusijie/Cocos2d-x-ReleaseNote#1]
I want to highlight a border of a sprite. I have a sprite type variable "m_sprImage" in which I want to highlight border.

I used below codes but not highlighting as per my game requirement.

TintTo\* action = TintTo::create(0.1f, 255, 0, 0);
//TintTo\* actionReverse = TintTo::create(5.5f, oldColor.r, oldColor.g, oldColor.b);;
TintTo\* actionReverse = TintTo::create(10.0f, 255, 255, 255);;
m_sprImage->runAction(Sequence::create(action, actionReverse, NULL));
m_sprImage->setOpacity(255);

**Comment by fusijie** [comment]
border of sprite? Do you means the rectangle or the the precise shape?
`tint` will influence the whole pictures.

**rectangle:** You need to use a special shader to process the border.
**precise shape:** You can refer to SpritePolygon to get the shape border. But it's not easy to handle it.

**Comment by Ravi1092** [comment]
du you have any sample code for precise shape?

**Comment by Ravi1092** [comment]
do you have any sample code for highlight border of a sprite using precise shape ?

**Comment by fusijie** [comment]
no... the algorithm of precise shape is complex. You can refer to SpritePolygon which may give you some inspiration.
